PyInstaller/hooks/hook-PyQt5.Qt.py

```diff
@@ -7,11 +7,15 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 #-----------------------------------------------------------------------------
 
-# When PyQt5.Qt is imported it implies a certain need for a few of the
-# other PyQt5 modules, but not all of them.
-hiddenimports = ['sip',
-                 'PyQt5.QtCore',
-                 'PyQt5.QtGui',
-                 'PyQt5.QtWidgets',
-                 'PyQt5.QtPrintSupport'
-                 ]
+# When PyQt5.Qt is imported it implies the import of all PyQt5 modules. See
+# http://pyqt.sourceforge.net/Docs/PyQt5/Qt.html.
+import os
+
+from PyInstaller.utils.hooks import get_module_file_attribute
+
+# Determine the name of all these modules by looking in the PyQt5 directory.
+hiddenimports = []
+for f in os.listdir(os.path.dirname(get_module_file_attribute('PyQt5'))):
```

Comment:
Good question. Basically, because `collect_submodules` prints sooooo much debug info (the entire PyQt5 subtree, which is huge), it becomes hard to follow what's happening. I'm considering replacing similar code in `hook-PyQt5.py` for the same reason.

Reply:
Thanks, done! (All changes made in the new branch.)